docs/source/workflow_inputs.rst

```diff
@@ -2963,6 +2966,7 @@ Each ``HeatPump`` should represent a single outdoor unit, whether connected to o
   .. [#] PanHeaterControlType choices are "continuous" or "defrost mode".
   .. [#] If PanHeaterControlType is "continuous", the pan heater will operate anytime the outdoor temperature is below 32F.
          If PanHeaterControlType is "defrost mode", the pan heater will only operate when the heat pump is in defrost mode and the outdoor temperature is below 32F.
+  .. [#] If BackupHeatingActiveDuringDefrost not provided, defaults to true if BackupType="integrated" and there is an attached distribution system, otherwise false.
```

Comment:
MSHPs may or may not be ducted, so we call out that dependency in the defaulting behavior.